backend/tests/test_models_endpoints.py

```diff
@@ -109,8 +119,16 @@ def test_dummy_model_only_valid_substances_are_present(
         }
     else:
         response.raise_for_status()
+        simulation_id = response.json()
+
+        response = client.get(f"/simulations/{simulation_id}/result")
```

Comment:
do we expect that this could turn out to be a flaky test? feels like it

have a function that will retry on "pending" response?